sklearn/metrics/_regression.py

```diff
@@ -434,6 +448,14 @@ def explained_variance_score(y_true, y_pred, *,
             Scores of all outputs are averaged, weighted by the variances
             of each individual output.
 
+    fix_when_y_true_is_constant : boolean, optional
+        Flag indicating if ``NaN`` and ``-Inf`` scores resulting from constant
+        data should be replaced with real numbers (``1.0`` if prediction is
+        perfect, ``0.0`` otherwise). Default is ``True``, a convenient settings
+        for model optimization procedures (e.g. grid search cross-validation).
+
+        .. versionadded:: 0.24
```

Comment:
```suggestion
    fix_when_y_true_is_constant : boolean, optional
        Flag indicating if ``NaN`` and ``-Inf`` scores resulting from constant
        data should be replaced with real numbers (``1.0`` if prediction is
        perfect, ``0.0`` otherwise). Default is ``True``, a convenient settings
        for model optimization procedures (e.g. grid search cross-validation).

        .. versionadded:: 1.1.0
```